src/definitions/modules/dropdown.js

```diff
@@ -4257,7 +4258,7 @@ $.fn.dropdown.settings.templates = {
     else {
       html += '<div class="text"></div>';
     }
-    html += '<div class="'+className.menu+'">';
+    html += '<div class="'+deQuote(className.menu)+'">';
```

Comment:
Note) `deQuote` converts `"` into `&quot;` which might break styling.
But I assume it is OK if class name contains `"` it is invalid and users should be aware of before production.

Reply:
No, it doesn't, because that only happens when a second parameter "encode" is provided. Default is false/undefined, so if only `deQuote()` is used the double quotes are simply getting removed. Only `deQuote('bla',true)` converts the double quotes into an htmlentity
https://github.com/fomantic/Fomantic-UI/blob/77bb2f281d60113cb684071d66d308670e4ff770/src/definitions/modules/dropdown.js#L4218-L4221